pkg/types/message_envelope.go

```diff
@@ -21,12 +21,16 @@ import (
 )
 
 const (
+	// Checksum value of the payload
+	Checksum      = "payload-checksum"
```

Comment:
Please lower case so it's not exported from the package.

Reply:
Sorry, I was thinking about exporting this value to be used on the `edgex-go` side for consistency.